packages/terra-badge/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Changed
+  * Update props description in props table.
```

Comment:
this PR has more changes than just prop description updates in the terra badge, can you also add logs for the src code changes

Reply:
Updated . Thanks
Commit Link -- b7717dd95fb57c5599a1f15df17a29e0baaabd4e